data/categories.txt

```diff
@@ -6447,9 +6447,10 @@ fa:مدرسه
 mr:शाळा
 
 amenity-shelter
-en:4Shelter
-ru:4Укрытие|навес
-bg:Подслон|Убежище|заслон
+en:4Shelter|refuge|lean-to
```

Comment:
Please, remove "lean-to". Due to _fancy_ category matching, it will add mess only.

Reply:
Is it because of a hyphen in it?

We're going to add a separate type for "Lean-to" in https://github.com/organicmaps/organicmaps/pull/7785
and this is how its really called...

Reply:
I.e. removing it in this PR is not a problem...
But its a big issue for https://github.com/organicmaps/organicmaps/pull/7785 as there are probably no other reasonable keywords to search for lean-tos while keeping it distinct from other shelters..

Reply:
Maybe we can track non-working hyphens in search in a separate issue? The search should take them into account. If a user types it, then it's meaningful and important.

Reply:
The last wrong result should be filtered out. Can you please create an issue?

Reply:
Wi-Fi is also not an appropriate keyword

Reply:
I've removed "lean-to" from here, @vng PTAL